Header header

# Board boot counter
uint32   boot_count

# Runtime (s)
duration running_time

# Temperatures (C)
float32 temp_cpu
float32 temp_chipset
float32 temp_system

# Voltages (V)
float32 vcore
float32 v2_5
float32 v3_3
float32 vbat
float32 v5
float32 v5sb
float32 v12

# Fan speeds (RPM)
uint32 fan_cpu

